### ISSUE TITLE
News summary page: make links clickable

### ISSUE THREAD

**Issue by raddick** [sciserver/SciServer-WordPress-Theme#1]
**tl;dr:** On the news summary page ([http://www.sciserver.org/news]http://www.sciserver.org/news), links do not show as links.

**Context:** When we push a new version, I add a News item pointing at the [Releases and Updates]http://test.sciserver.org/support/updates/#overview1p9p4 page. When you go to that specific news item, the link is available for clicking. But these news items are designed to be short, with the real information coming in the Releases & Updates page. So it would be great to have the link also show up in the summary page, so people don't have to click on the individual news item to follow the link. (If, OTOH, there's a usability reason to make them not-clickable links, leave a comment and let's discuss.) Thanks!

**Comment by bonbons0220** [comment]
Probably will be superseded by new website design. If not, reopen.
